doc/APIGuide.md

```diff
@@ -57,28 +57,28 @@ model.getMassCenter(state); // Need to know pose of model.
+3. Cache variables allow saving of (state-dependent) calculations that may be used multiple times, to improve computational efficiency. For example, computing the path length of a muscle with several intermediate points and wrapping over obstacles is expensive. Therefore, a path component itself caches its length so that the muscle (or any other component) that needs the length in its computations does not reevaluate it unnecessarily. The validity of the path length cache variable is dependent on the generalized coordinates--as long as these generalized coordinates do not change, the path length is valid. OpenSim employs categories of dependency known as realization Stages (see Simbody User Guide, Section 2.4 for details) to manage the validity of cache variables. Any changes to the generalized coordinates invalidate (and wipe out) any cache variables associated with the Position Stage and all other subsequent (Velocity, Dynamics, Acceleration) stages. OpenSim automatically invalidates cache variables to support the correctness of simulations. Manual cache management is highly prone to errors leading to “stale” variables and potentially incorrect physics. Sadly, while results may appear plausible they are completely invalid and sometimes impossible to detect and debug. If you need to cache for performance, use OpenSim’s facilities.
 
 ### Models are composed of Components {#modelscomposed}
 A Component encapsulates (all or part of the) system dynamics (of a model or element of a model) and provides (computes) values of interest. A component adds its dynamics and allocates necessary resources (state, cache, and other variables). It defines parameters (see Properties below) that specify its behavior/function and defines dependencies on any other Components (see Sockets below) via its Inputs and Outputs.
 
-Components of a [Model](@ref OpenSim::Model) form a rooted directed tree topology of ownership (Components own and know about their subcomponents not the other way around). A root component (the top level Model) contains all the necessary Components to define the System. Components in the tree are uniquely identified by their full path name from the root (like a file path)
+Components of a [Model](@ref OpenSim::Model) form a rooted directed tree topology of ownership (Components own and know about their descendents). A root component (the top level Model) contains all the necessary Components to define the System. Components in the tree are uniquely identified by their full path name from the root (like a file path)
 
 A submodel (e.g., a subassembly in CAD) forms its own branch of a Model. This might be a Device, a Leg model or even a FullBody model.
```

Comment:
Is there a reason to throw in the word "submodel" here? Can we just stick with a (top-level) model can be composed of other models? Down the line we may want to use submodel to actually indicate something more concrete.

Reply:
Thanks for the feedback @chrisdembia need to resolve the "submodel" reference here and then this can be merged.

Reply:
The "submodel" term was not thrown in. It is not meant to be a "top-level model". It is a grouping of subcomponents that represents something meaningful to the user. This can be a device, a limb, ...  I don't see why it is confusing/offensive in this context.  I am open to suggestions for other words that fit the context of the paragraph.

Reply:
My suggestion would be to just use Model (which is already well defined to users at all levels), and not introduce the word "Submodel" here. Earlier the document implieed that a Model can be composed of Models, this can be reworded as an example application of this design.

Reply:
I don't believe that:
*A **Model** (e.g., a subassembly in CAD) forms its own branch of a Model.* doesn'y make much sense. We can rewrite the sentence if "submodel" is a deal breaker here.

Reply:
Perhaps:
A branch in the ownership tree of a Model can be a complete subassembly representing a Device, a Leg or another FullBody Model.

Reply:
"Model" does not work in this context. "A *Model* (e.g., a subassembly in CAD) forms its own branch of a Model." ? 
Perhaps:
A Component of a Model can be a complete subassembly representing a Device, a Leg or another FullBody Model.